contracts/token/ERC721/extensions/IERC721Rental.sol

```diff
@@ -9,14 +9,14 @@ import "../../../utils/introspection/IERC165.sol";
 /// Defines the interface that a rental agreement contract should support to be used by
 /// `IERC721Rental`.
 interface IERC721RentalAgreement is IERC165 {
-    /// Function called at the end of `IERC721Rental.setRentalAgreement` to replace an
-    /// existing rental agreement contract of a token.
+    /// Function called at the end of `IERC721Rental.setRentalAgreement` when an agreement
+    /// is being unlinked from a token.
     ///
     /// @dev Allows the agreement to cancel the change by reverting if it deems it
     /// necessary. The `IERC721Rental` is calling this function, so all information needed
-    /// can be queried through the `msg.sender`. This event is not called if a rental agreement
-    /// is not in progress.
-    function afterRentalAgreementReplaced(uint256 tokenId) external;
+    /// can be queried through the `msg.sender`. This event is only called if a rental is
+    /// not in progress, otherwise the `IERC721Rental` does not allow an agreement change.
```

Comment:
I think it's if a rental is in progress?

Reply:
no it's correct
because otherwise the erc721 contract will generate an error before even trying to call the event

Reply:
Ah I see, I think there could be confusion between "is rental is in progress" and "a rental agreement has been set up" 
Here it will be called only if "a rental agreement has been set up"  but not if "is rental is in progress"

Reply:
Well since it is called on an agreement, I would assume it is obvious that the agreement must exist :-)

Reply:
maybe instead of "a rental is not in progress" we could change by there is no renters?

Reply:
I find it less clear.
"The event is called when a rental agreement is removed, which cannot happen during a rental."?

Reply:
what was unclear to me is the definition of "in progress"